### PR TITLE
natashafir task html-css-popup

### PR DESCRIPTION
**Pull request by natashafir**
### hooli-style-popup

[Demo](https://natashafir.github.io/hooli-style-popup) |
[Code base](https://github.com/natashafir/hooli-style-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/natashafir/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link href="./css/style.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="nav">
```
Comment (madmaxWMFU) [inline submissions/natashafir/html-css-popup/index.html:13]: Use a special tag `nav` for navigation links.

File: submissions/natashafir/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        </li>
+        <li class="nav-item">
+            <a href="#" class="social__item user" tabindex="0">
+                <img src="./img/user.png" alt="">
+            </a>
+        </li>
+    </ul>
+</div>
+</body>
+</html>
```
Comment (madmaxWMFU) [inline submissions/natashafir/html-css-popup/index.html:80]: Add an empty line.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Excellent work!
See some comments below

File: submissions/natashafir/html-css-popup/css/style.css [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+.popup-item {
+    padding: 6px;
+    opacity: 0.7;
+}
+
+.popup-item:hover, .more-label:hover {
+    opacity: 1;
+}
+
+li:focus, input:focus + label {
```
Comment (madmaxWMFU) [inline submissions/natashafir/html-css-popup/css/style.css:61]: Avoid styling by tag. Except for the assignment of global base styles.

File: submissions/natashafir/html-css-popup/css/style.css [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+    display: none;
+}
+
+.popup-menu::-webkit-scrollbar {
+    width: 3px;
+}
+
+.popup-menu::-webkit-scrollbar-thumb {
+    background-color: var(--accent-color);
+}
```
Comment (madmaxWMFU) [inline submissions/natashafir/html-css-popup/css/style.css:215]: Add an empty line.

File: submissions/natashafir/html-css-popup/css/style.css [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+.nav {
+    color: var(--white-color);
+    float: right;
+    display: flex;
+    align-items: center;
+}
+
+#popup, #more-btn {
```
Comment (madmaxWMFU) [inline submissions/natashafir/html-css-popup/css/style.css:122]: Avoid styling by id.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]

File: submissions/natashafir/html-css-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+        <li class="nav-item"><a href="#" class="social__item img" tabindex="0">Images</a></li>
+        <li class="nav-item">
+            <div class="social__item menu">
+
+                <input type="checkbox" id="popup">
+                <label for="popup" class="popup-label"></label>
+
+                <div class="popup-menu">
+                    <a href="#" class="popup-item">
+                        <img src="./img/html.png" alt="">
```
Comment (MaxKalashnyk) [inline submissions/natashafir/html-css-popup/index.html:25]: It's not a blocker, but don't forget about setting the alternative text for your images in the `alt` attribute

Comment (natashafir) [reply submissions/natashafir/html-css-popup/index.html:25]: Thank you.

File: submissions/natashafir/html-css-popup/css/style.css (hunk 5 above)
Comment (natashafir) [reply submissions/natashafir/html-css-popup/css/style.css:122]: Thank you!

**Review by miroslav-kolomiets — COMMENTED** [review]
Looks great! But in addition to the above, it is better to stick to the same style in the files, and do not add multiple blank lines or unnecessary blank lines. As well as remember to add a blank line at the end of each file.

**Review by madmaxWMFU — APPROVED** [review]

**Review by MaxKalashnyk — APPROVED** [review]

**Comment by natashafir** [comment]
> Looks great! But in addition to the above, it is better to stick to the same style in the files, and do not add multiple blank lines or unnecessary blank lines. As well as remember to add a blank line at the end of each file.

Thanks for review. Done.

**Review by miroslav-kolomiets — APPROVED** [review]

**Pull request merged**